src/jdk.jartool/share/classes/sun/security/tools/jarsigner/Resources.java

```diff
@@ -178,6 +179,7 @@ public class Resources extends java.util.ListResourceBundle {
         {"key.bit.disabled", "%d-bit key (disabled)"},
         {"key.bit.eccurve.disabled", "%1$d-bit %2$s key (disabled)"},
         {"unknown.size", "unknown size"},
+        {"nonexistent.entries.found", "Nonexistent signed entries detected. See details in -verbose output."},
```

Comment:
For the second sentence, in other warning messages, we say "Re-run jarsigner with the -verbose option for more details." Perhaps we should be consistent?

Reply:
Actually, I see the output will also contain the message "Re-run with the -verbose and -certs options for more details." so I take back my comment above.

Reply:
For this summary message, I suggest we be a bit more descriptive like in other messages:

"This jar contains signed entries for files that do not exist. See the -verbose output for more details."

Reply:
Updated. Thanks.